docs/index.md

```diff
@@ -94,9 +94,89 @@ If the above doesn't help, please [submit an issue](https://github.com/fastlane/
 
 Currently, _fastlane_ is officially supported to run on macOS. Support for other operating systems is limited and untested at this point in time. _fastlane_ uses system APIs that may not be implemented on other platforms, for example, we use the Ruby `fork` method for sub-process management, which isn't available on Windows.
 
-## Special Thanks
-
-Thanks to all [contributors](https://github.com/fastlane/fastlane/graphs/contributors) for extending and improving _fastlane_.
```

Comment:
Maybe this should not be fully removed - there are many more people in the contributor list that deserve to be thanked as well, right?

Reply:
Oh good point, do you mind submitting a PR for this?

Reply:
Just edit the file directly or is this also generated/done via a script?

Reply:
Done